com.unity.multiplayer.mlapi/Editor/NetworkObjectEditor.cs

```diff
@@ -27,7 +27,7 @@ public override void OnInspectorGUI()
         {
             Initialize();
 
-            if (!m_NetworkObject.IsSpawned && m_NetworkObject.NetworkManager != null && m_NetworkObject.NetworkManager.IsServer)
+            if (EditorApplication.isPlaying && !m_NetworkObject.IsSpawned && m_NetworkObject.NetworkManager != null && m_NetworkObject.NetworkManager.IsServer)
```

Comment:
Wouldn't it be safer to do:
`if (EditorApplication.isPlaying && m_NetworkObject && !m_NetworkObject.IsSpawned && m_NetworkObject.NetworkManager != null && m_NetworkObject.NetworkManager.IsServer)`
e.g. checking that the m_NetworkObject is set ?